src/components/Button/Button.stories.tsx

```diff
@@ -1,24 +1,80 @@
 import React from 'react';
-import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Button } from './Button';
+import { Story, Meta } from '@storybook/react';
 
-import Button from './Button';
-
-// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
 export default {
   title: 'Components/Button',
   component: Button,
-} as ComponentMeta<typeof Button>;
+  argTypes: {
+    disabled: {
+      control: { type: 'boolean' },
+    },
+  },
+  parameters: { actions: { argTypesRegex: '^on.*' } },
+} as Meta;
+
+const Template: Story = ({ ...args }) => <Button {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {
```

Comment:
I thought that we agreed on having Default button as our main yellow button, but this seems to be Default Material-UI component. But let's ask @tobiastykvart, maybe I misunderstood